.circleci/config.yml

```diff
@@ -720,6 +720,14 @@ jobs:
       - DEPLOY_DOWNTIME: 900 #15M
     steps: *deploy_steps
 
+  deploy_next:
+    executor: infrastructure_container_unstable
+    <<: *infrastructure_config
+    environment:
+      - DEPLOY_ENV: next
```

Comment:
I see hard forks triggered at height 1 and 2 (https://github.com/aeternity/infrastructure/commit/6fe6981126ddfd5dd3718b012d8fbd4d9d61e9cc#diff-964fd20d87bd60f6c7d0c03a5c78e506R26) and network id ae_next - ok.